pkg/neotest/basic.go

```diff
@@ -158,18 +162,31 @@ func (e *Executor) DeployContractBy(t testing.TB, signer Signer, c *Contract, da
 			stackitem.NewByteArray(c.Hash.BytesBE()),
 		}),
 	})
-
 	return tx.Hash()
 }
 
 // DeployContractCheckFAULT compiles and deploys a contract to the bc using the validator
 // account. It checks that the deploy transaction FAULTed with the specified error.
 func (e *Executor) DeployContractCheckFAULT(t testing.TB, c *Contract, data any, errMessage string) {
+	e.trackCoverage(t, c)
+
 	tx := e.NewDeployTx(t, e.Chain, c, data)
 	e.AddNewBlock(t, tx)
 	e.CheckFault(t, tx.Hash(), errMessage)
 }
 
+// This switches on coverage tracking for provided script if `go test` is running with coverage enabled.
+func (e *Executor) trackCoverage(t testing.TB, c *Contract) {
```

Comment:
It looks good when comment starts with the method name even for non-exported methods, i.e. `trackCoverage switches on coverage tracking...`